### PR TITLE
feat(ingest): looker - support for simple column level lineage

### PR DESCRIPTION
**Pull request by shirshanka**
Adds very simple column level lineage for looker
Table -> View
and 
View -> Explore.

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py [hunk 1]
```diff
@@ -746,20 +756,52 @@ def _to_metadata_events(  # noqa: C901
...
+                view_name_to_urn_map[view_name] = view_urn
+            if config.extract_column_level_lineage:
+                for field in self.fields or []:
+                    if (
+                        field.upstream_field
+                        and len(field.upstream_field.split(".")) >= 2
+                    ):
+                        (view_name, field_path) = field.upstream_field.split(".")[
+                            0
+                        ], ".".join(field.upstream_field.split(".")[1:])
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py:786]: nit: use split(maxsplit=1)

File: metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py [hunk 2]
```diff
@@ -586,6 +593,19 @@ def _get_fields(
             native_type = field_dict.get("type", "string")
             description = field_dict.get("description", "")
             label = field_dict.get("label", "")
+            upstream_field = None
+            if type_cls == ViewFieldType.DIMENSION and extract_column_level_lineage:
+                if field_dict.get("sql") is not None:
+                    upstream_field_match = re.match(
+                        r"^.*\${TABLE}\.(.*)$", field_dict["sql"]
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py:600]: nit: add a comment explaining what this regex does

**Review by hsheth2 — APPROVED** [review]
two nits, but neither is a show stopper so they can be done in a follow up

**Comment by shirshanka** [comment]
thx @hsheth2

**Pull request merged**